src/svxlink/contrib/module_site_status/ModuleSiteStatus.conf

```diff
@@ -0,0 +1,80 @@
+[ModuleSiteStatus]
+NAME=SiteStatus
+PLUGIN_NAME=Tcl
+
+
+# ID field is intentionally long and repetative, the module is not intended to have user
+# interaction, it is intended to sit in the background and monitor signals and send
+# alerts when something has triggered a sensor.
+
+# TIMEOUT is intentionally very short (1 sec).  the module is not intended to have user
+# interaction, it is intended to sit in the background and monitor signals and send
+# alerts when something has triggered a sensor.
```

Comment:
You left the comments about ID and TIMEOUT in there but I'll fix that for you so that we can get this PR merged